src/main/java/io/ballerina/plugins/idea/sdk/BallerinaSdk.java

```diff
@@ -0,0 +1,27 @@
+package io.ballerina.plugins.idea.sdk;
+
+public class BallerinaSdk {
```

Comment:
can convert this into a Java record

Reply:
Checkstyle plugin does not work when converted to a record.

Reply:
As discussed offline, lets update the checkstyle configs via a separate PR